apps/api/e2e/tests/competition.test.ts

```diff
@@ -261,7 +260,7 @@ describe("Competition API", () => {
     const buyTradeResponse = await agentClient.executeTrade({
       reason: "testing create comp with trading constraints",
       fromToken: config.specificChainTokens.eth.usdc,
-      toToken: VISION_TOKEN, // low 24h volume
+      toToken: config.specificChainTokens.svm.sol,
```

Comment:
@joewagner i think we should remove `VISON_TOKEN` as part of #881. these tests started to fail because the token somehow became "tradable" and didnt fail the test.